### DISCUSSION TITLE
很喜欢你做的开源项目，尤其是园丁鸟和快传哈哈

### DISCUSSION THREAD

**Discussion by Leizhenpeng** [NXY666/swift-share#9] [讨论]
看到 issue 说最近你在找工作哈哈

可以聊聊不，感觉我们志趣相投哈哈

（：我已经创业了哈哈，有不少搞事情机会！

@NXY666

**Comment by NXY666** [comment]
感谢你的喜欢。我已经找到工作啦！

之后还会继续开发项目，不过工作后可能就没法那么频繁地开发了。

**Reply by Leizhenpeng** [reply]
可以邮件沟通一下，也许可以作为 side project 哈哈， (: laolai@forkway.cn
